led-display/client.ts

```diff
@@ -116,11 +118,19 @@ namespace modules {
         //% weight=98
         setPixels(pixels: Buffer) {
             if (!pixels) return;
+            this._localPixels = pixels;
+        }
 
+        show() {
```

Comment:
is show called at a regular interval automatically?

Reply:
no.